onnx/onnx.in.proto

```diff
@@ -287,6 +287,9 @@ message GraphProto {
   // Initializers (see above) stored in sparse format.
   repeated SparseTensorProto sparse_initializer = 15;
 
+  // Initializers for tensor values in the form of graphs.
+  repeated GraphProto graph_initializer = 16;
```

Comment:
don't really understand the scenario of having a "Graph" as initializers of a model. Do you mind to elaborate that in our gitter room. https://gitter.im/onnx/Infra